packages/forma-36-website/src/components/Navigation.js

```diff
@@ -1,8 +1,8 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'gatsby';
-/** @jsx jsx */
```

Comment:
Not sure why this isn't needed anymore or why it was added in the first place. I think it was required after upgrading something.

Reply:
yeah I removed and then nothing broke so I kept the change hahaha